castai/resource_node_configuration.go

```diff
@@ -161,9 +161,21 @@ func resourceNodeConfiguration() *schema.Resource {
 						"key_pair_id": {
 							Type:             schema.TypeString,
 							Optional:         true,
-							Description:      "AWS key pair ID to be used for provisioned nodes. Has priority over ssh_public_key",
+							Description:      "AWS key pair ID to be used for CAST provisioned nodes. Has priority over ssh_public_key",
 							ValidateDiagFunc: castval.ValidKeyPairFormat(),
 						},
+						"volume_type": {
+							Type:             schema.TypeString,
+							Optional:         true,
+							Description:      "AWS EBS volume type to be used for CAST provisioned nodes",
+							ValidateDiagFunc: castval.ValidEc2VolumeType(),
```

Comment:
```suggestion
							ValidateDiagFunc: validation.ToDiagFunc(validation.StringInSlice([]string{"gp3", "io1", "io2"}, true)),
							
```

WDYT?

Reply:
Makes sense! That was I was looking for.